assembly/std/index.ts

```diff
@@ -20,12 +20,17 @@ export function print(message: string): void {
  *
  * Note: arguments serialization is to be handled by the caller and the callee.
  *
- * @param at -
- * @param functionName -
- * @param args -
- * @param coins -
+ * @param at - The address of the contract.
+ * @param functionName - The name of the function to call in that contract.
+ * @param args - The serialized arguments to send to the function to call.
+ * @param coins - The max number of coins to be used by the call. (If not enough, the call will stop !)
```

Comment:
the argument coins refer to the tokens that we send to the contract. Their aren't the price you are paying for the execution

Reply:
Fixed that, it is more likely that this arguments describes coins to pass to some kind of "payable" function, can you confirm that @BatiGencho @Thykof @Ben-Rey ?

Reply:
probably yes